ReactWindows/ChakraBridge/ChakraHost.cpp

```diff
@@ -1,8 +1,13 @@
-﻿#include "pch.h"
+﻿#include <stdint.h>
+#include "pch.h"
```

Comment:
Looks like <> comes after "" in other places.

---
In reply to: [103544307](https://github.com/ReactWindows/react-native-windows/pull/1017#discussion_r103544307) [](ancestors = 103544307)